### ISSUE TITLE
GR00T-N1.6 (VLA): no inference / open-loop-eval / deployment path after SFT training — Megatron checkpoint has no reverse converter and no inference entry-point

### ISSUE THREAD

**Issue by ShawRong** [baidu-baige/LoongForge#92]
## Summary

After finetuning a VLA model (`--model-name groot_n1_6`, `--training-phase sft`), there is no shipped/documented way to **use the trained checkpoint** — no inference entry-point, no open-loop eval, and no path to the official Isaac-GR00T evaluation/deployment toolchain. The training side is complete, but "what do I do with the checkpoint" is missing.

## What's there vs. missing

- Training saves a **Megatron-format** checkpoint, e.g.:
  ```
  <save>/iter_0010000/mp_rank_00/{model_optim_rng.pt, distrib_optim.pt}
  ```
- There is **no gr00t/VLA inference or eval script** in LoongForge. (`grep -ri eval/infer` only surfaces LLM / multimodal / Megatron-generic paths, nothing for `groot_n1_6` / `pi05`.)
- A **forward** converter exists — `examples/groot_n1.6/checkpoint_convert/convert_gr00tn1.6_hf_to_torch.sh` → `tools/convert_checkpoint/pi05/convert_hf_to_torch.py` (HF → Megatron). But there is **no reverse `torch → HF` converter** for `pi05`/`groot_n1_6`.
- The official **Isaac-GR00T** eval/deploy tools (`gr00t/eval/open_loop_eval.py`, `gr00t/eval/run_gr00t_server.py`, `gr00t/eval/real_robot/`) all require an **HF-format** GR00T model — which LoongForge can't currently produce from a trained checkpoint.
- The model itself **already implements inference**: `Gr00tN1d6.get_action()` and `Gr00tN1d6ActionHead.get_action()` (`loongforge/models/embodied/groot_n1_6/modeling_groot.py:1005` / `:563`, flow-matching Euler integration). So the capability exists — only the loading/entry-point + action unnormalization wiring is missing.

Net effect: a user can train `groot_n1_6` end-to-end but cannot run the result on data or a robot without writing their own loader + Megatron→HF mapping + unnormalization.

## Request

Provide one (ideally both) of:

1. **An inference / open-loop-eval entry-point for VLA models** that:
   - loads a trained Megatron checkpoint (`--load <ckpt>`),
   - builds the model + processor (reusing `make_gr00t_n1d6_pre_post_processors`),
   - runs `get_action()` on dataset samples (or a live obs),
   - un-normalizes actions using the dataset/embodiment stats,
   - and (for open-loop) reports predicted-vs-ground-truth action error.

2. **A `torch → HF` reverse converter** for `pi05` / `groot_n1_6` (inverse of `convert_*_hf_to_torch.py`), so LoongForge-trained checkpoints can be consumed by the official Isaac-GR00T `open_loop_eval.py` / `run_gr00t_server.py` / real-robot deployment.

Plus a short section in the VLA tutorial (`docs/.../vla_tutorial/`) covering the **post-training** step (currently the tutorials stop at training).

## Environment / repro

- Commit: `3bba602`
- Trained `groot_n1_6` SFT, checkpoint at `<save>/iter_0010000/mp_rank_00/model_optim_rng.pt` (Megatron `--ckpt-format torch`).
- Looking for: "how do I run inference / evaluate / deploy this?" — no current answer in the repo or docs.
